ui/webui/test/machine_install.py

```diff
@@ -62,18 +63,52 @@ def _create_disk_image(self, size, image_path=None, quiet=False):
         self._execute(f"qemu-img create -f qcow2 {quiet} {image_path} {size}G")
         return image_path
 
+    def _wait_http_server_running(self, port):
```

Comment:
What is wrong with the existing `wait_for_cockpit_running('127.0.0.1', http_port)` [code](https://github.com/cockpit-project/bots/blob/000d0e34b42a978ad13b6ab55219fbff8296eeff/machine/machine_core/machine.py#L320) ? You can't access that API from anaconda, due to the different Machine class?

Reply:
Yeah, the name is confusing, it should really be called `wait_for_http_port`..

Reply:
Argh, my bad :see_no_evil: